source/reference/verification.txt

```diff
@@ -16,117 +16,131 @@ Verify Data Transfer
+       verifier, which runs a series of verification checks on
+       the source and destination clusters to confirm that the
+       migration was successful. This is the preferred
+       verification method for deployments that meet the
+       :ref:`requirements <c2c-verify-embedded-limitations>`.
+
+       When the ``mongosync`` process starts, it prompts the
+       user with a disclaimer about the embedded verifier. You
+       must accept the disclaimer, or if you have already read
+       and acknowledged it, start ``mongosync`` with the
+       :option:`--acceptDisclaimer` option to use the verifier.
+
+       The verifier doesn't check every aspect of a migration:
+
+       - To verify sync on unsupported namespaces and sharded
+         clusters, use a different verification method.
+       - To verify index sync, use the :ref:`Index Comparison
+         <c2c-index-comparison>` method. 
+       - To verify metdata sync, use the :ref:`Metadata
+         Comparison <c2c-metadata-comparison>` method.
+
+   * - :ref:`Document Counts <c2c-verify-doc-counts>`
+     - The most basic method of verification is to compare the
+       number of documents in each synced collection on the
+       source cluster to the number on the destination cluster.
+       
+       Before you can verify data transfer with this method,
+       ``mongosync`` must be in the ``COMMITTED`` state.
+
+       This method only verifies a successful sync when run
+       against clusters with insert-only workloads.
+
+   * - :ref:`Hash Comparison <c2c-verify-hash-comp>`
+     - You can verify sync by comparing MD5 hashes of
+       collections synced from the source cluster to the
+       destination cluster.
+
+       Before you can verify data transfer with this method,
+       ``mongosync`` must be in the ``COMMITTED`` state.
+
+       While hash comparison ensures that the destination
+       cluster has received all changes from the source, the
+       :dbcommand:`dbHash` command locks the cluster, preventing
+       additional writes until it completes.
+
+       Hash comparison is **not** possible with sharded
+       clusters. It also does not work with standalone servers
+       and replica sets that use MongoDB 4.4 or earlier
+       releases, since the document field order can vary.
+
+   * - Document Comparison
+     - You can verify sync by comparing documents on the source
+       and destination clusters. 
+
+       Write a script that queries collections on the source
+       cluster and then checks that the correct documents,
+       indexes, collections, metadata, and views exist with the
+       same values on the destination cluster.
+
+       Before you can verify data transfer with this method,
+       ``mongosync`` must be in the ``COMMITTED`` state.
+
+   * - .. _c2c-index-comparison: 
+
+       Index Comparison
+     - To verify the transfer of indexes, run the
+       :method:`db.collection.getIndexes` method on the source
```

Comment:
thank you for adding this section! I am confirming all looks good with the engineering team, my initial read is this looks good

Reply:
Looks good!